app/policies/organization_policy.rb

```diff
@@ -18,4 +18,10 @@ class TranslateScope < Scope
   scope :index, :show, :versions, :version, with: ReadScope
   scope :update, :update_links, :destroy, :destroy_links, with: WriteScope
   scope :translate, with: TranslateScope
+
+  def linkable_projects
+    # TODO: Is this really right? This lets me stick any
+    # project in my organization. What if they don't want to?
+    policy_for(Project).scope_for(:show)
```

Comment:
I think this should have been using scope_for update?

Reply:
Looks like it was / is `can_be_linked :project, :scope_for, :update, :user` https://github.com/zooniverse/Panoptes/blob/040204f46f7c8818e2beb37e433e5507a50b5917/app/models/organization.rb#L32

Reply:
Excellent, thanks for validating the entire refactor for me. You've gotten confused by the old code. 

`Organization.can_be_linked(:project)` is used by `Organization.link_to_resource(project)`, which is called by *Projects*Controller when you pass in a `links: {organization: 123}` there. 

This refactor would normally move that to `ProjectPolicy.linkable_organizations`. Except, `organization` isn't in the permitted links: 

* https://github.com/zooniverse/Panoptes/blob/master/config/routes.rb#L104
* https://github.com/zooniverse/Panoptes/blob/master/app/schemas/project_update_schema.rb#L124
* https://github.com/zooniverse/Panoptes/blob/master/app/schemas/project_create_schema.rb#L112

You *can* link projects at the organization side (https://github.com/zooniverse/Panoptes/blob/master/config/routes.rb#L97). In that case the controller would call `Project.link_to_resource(organization)` which uses `Project.scope_for(:show)` because there is no explicit `can_be_linked` defined.

Reply:
Oh, but please double check that I've got the above reasoning about the existing workings of JsonApiController, RelationManager and Linkable correct!

Reply:
Good shout - indeed I had this backwards and always get confused by this. Indeed we did get it wrong originally when settting up orgs. The `can_be_linked` permissions should be on the other side of the relation and is so on other models. 

Would removing the `default_link_to_scope(user)` in `linkable.rb` surface the erroneous model link setups?